src/python/pants/testutil/test_base.py

```diff
@@ -688,37 +688,6 @@ def create_library(
         )
         return self.target(f"{path}:{name}")
 
-    def create_resources(self, path, name, *sources):
-        """
-        :API: public
-        """
-        return self.create_library(path=path, target_type="resources", name=name, sources=sources,)
```

Comment:
We need to get rid of `self.create_library` because it requires V1. Generally, I've found that `self.add_to_build_file` works well instead.